packages/table/src/regions.ts

```diff
@@ -467,6 +467,16 @@ export class Regions {
         return true;
     }
 
+    public static isRegionValidForTable(region: IRegion, numRows: number, numCols: number) {
+        if (region.rows != null && (region.rows[0] >= numRows || region.rows[1] >= numRows)) {
```

Comment:
Guh. 

Somehow I parsed your thing as asking why I was checking if it was < 0 and that caused me to believe that is what I had done. As opposed to doing the opposite. It's a little disturbing my memory is so malleable. 

We don't need it because you can't have negative regions, but I can add it if it's necessary.